mypy/test/teststubgen.py

```diff
@@ -97,6 +104,14 @@ def test_packages_found(self) -> None:
                         os.path.join("pack", "b.py"),
                     },
                 )
+                ignored_files = {os.path.relpath(mod.path or "FAIL") for mod in ignored_py_mods}
```

Comment:
what is `"FAIL"`?

Reply:
In fact I wrote in the same way as `files` initialized here. 
https://github.com/python/mypy/blob/1104810a2b0a283f9452ea8b343d2ce7e11ddf5f/mypy/test/teststubgen.py#L98

StubSource.path may be None, so I guess the original code intend to express failure of module finding and also avoiding error which caused by passing None to os.path.relpath.
Maybe "NOT_FOUND" or just skipping mod without mod.path like below is better. Which do you think is better?
```
{os.path.relpath(mod.path) for mod in ignored_py_mods if mod.path is not None}
```